docs/source/getting-started/installation.rst

```diff
@@ -323,7 +323,7 @@ By default, the provisioned nodes will be in the root `compartment <https://docs
       compartment_ocid: ocid1.compartment.oc1..aaaaaaaa......
 
 
-Lambda Cloud
+Lambda cloud
```

Comment:
We should keep the cloud names capitalized. Can we add the entire cloud name to the exception list? (as in `'Lambda Cloud'`, `'Cudo Computer'`, `'Samsung Cloud Platform'`)